tables/rx_antibodies/ikemura21-mab.csv

```diff
@@ -1,3 +1,4 @@
 ref_name,rx_name,ab_name
-Ikemura21,REGN10987+REGN10933,Imdevimab+Casirivimab
+Ikemura21,REGN10987+REGN10933,Imdevimab
```

Comment:
you need change `REGN10987+REGN10933` to `Imdevimab+Casivirimab`. The rx_name `Imdevimab+Casivirimab` is used in rx_potency.csv